src/Mono.Android/Test/Mono.Android-Tests.csproj

```diff
@@ -116,17 +116,5 @@
       <Project>{CB2335CB-0050-4020-8A05-E9614EDAA05E}</Project>
       <Name>TestRunner.NUnit</Name>
     </ProjectReference>
-    <ProjectReference Include="..\..\Xamarin.Android.Build.Tasks\Xamarin.Android.Build.Tasks.csproj" Condition=" '$(XAIntegratedTests)' == 'True' ">
-      <Name>Xamarin.Android.Build.Tasks</Name>
-      <Project>{3F1F2F50-AF1A-4A5A-BEDB-193372F068D7}</Project>
-      <Private>False</Private>
-      <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
-    </ProjectReference>
```

Comment:
Could we remove this reference? I think the purpose of this is to be sure we have working xabuild.

Reply:
Ah, I see, we don't differentiate by `XAIntegrationTests` anymore